libc/crt0.S

```diff
@@ -10,9 +10,9 @@
 
 // This is the program entry point
 
-	.global entry
+	.global _start
 
-entry:
+_start:
 	jmp _startup
```

Comment:
Plus the fact that `_start` could be confused with C `_startup`...

Reply:
On second thought, I think this jump is not very useful.  I will remove it.

Reply:
Yeah, well done, it solves the 'confusion point'.